### ISSUE TITLE
Invisible UAC on Windows.Graphics.Capture API

### ISSUE THREAD

**Issue by GaryCraft** [LizardByte/Sunshine#3487]
### Is there an existing issue for this?

- [x] I have searched the existing issues

### Is your issue described in the documentation?

- [x] I have read the documentation

### Is your issue present in the latest beta/pre-release?

This issue is present in the latest pre-release

### Describe the Bug

When an UAC prompt is given the stream freezes until resolved, and it is invisible

### Expected Behavior

UAC prompt is visible and interactable as always using the Desktop duplication API

### Additional Context

I was having issues with the Xbox gamebar (it was invisible), and found in this repo that a beta usage of a different API for capturing the desktop had resolved this issue in the latest prelease, so I installed it and ran sunshine manually instead of using the service as described, this did resolve the invisible game bar issue, however it seems that now I cannot see windows UAC prompts

### Host Operating System

Windows

### Operating System Version

Windows 11 Pro Build 26100

### Architecture

amd64/x86_64

### Sunshine commit or version

v2024.1223.230344

### Package

Windows - installer (recommended)

### GPU Type

NVIDIA

### GPU Model

Quadro P1000 PNY 4G

### GPU Driver/Mesa Version

566.03

### Capture Method

Windows.Graphics.Capture (Windows)

### Config

```Shell
capture = wgc
```

### Apps

```json

```

### Relevant log output

```shell
[2024-12-26 02:45:17.359]: Info: New streaming session started [active sessions: 1]
[2024-12-26 02:45:17.554]: Info: nvprefs: No need to modify application profile settings
[2024-12-26 02:45:17.554]: Info: nvprefs: No need to modify global profile settings
[2024-12-26 02:45:17.590]: Info: CLIENT CONNECTED
[2024-12-26 02:45:17.872]: Info: Audio capture format is [F32 48000 2.0]
[2024-12-26 02:45:17.873]: Info: Opus initialized: 48 kHz, 2 channels, 512 kbps (total), LOWDELAY
[2024-12-26 02:45:17.969]: Info: 
Device Description : NVIDIA Quadro P1000
Device Vendor ID   : 0x000010DE
Device Device ID   : 0x00001CB1
Device Video Mem   : 4004 MiB
Device Sys Mem     : 0 MiB
Share Sys Mem      : 12252 MiB
Feature Level      : 0x0000B100
Capture size       : 1920x1080
Offset             : 0x0
Virtual Desktop    : 1920x1080
[2024-12-26 02:45:17.970]: Info: Active GPU has HAGS enabled
[2024-12-26 02:45:17.970]: Info: Using realtime GPU priority
[2024-12-26 02:45:17.970]: Info: 
Colorspace         : DXGI_COLOR_SPACE_RGB_FULL_G22_NONE_P709
Bits Per Color     : 8
Red Primary        : [0.679688,0.30957]
Green Primary      : [0.206055,0.693359]
Blue Primary       : [0.151367,0.0546875]
White Point        : [0.313477,0.329102]
Min Luminance      : 0.5 nits
Max Luminance      : 270 nits
Max Full Luminance : 270 nits
[2024-12-26 02:45:18.069]: Info: Creating encoder [hevc_nvenc]
[2024-12-26 02:45:18.070]: Info: Color coding: SDR (Rec. 601)
[2024-12-26 02:45:18.070]: Info: Color depth: 8-bit
[2024-12-26 02:45:18.070]: Info: Color range: MPEG
[2024-12-26 02:45:18.151]: Info: NvEnc: created encoder HEVC P1 two-pass
```

**Comment by ReenigneArcher** [comment]
It won't work until wgc works with the service.

**Comment by Nonary** [comment]
> It won't work until wgc works with the service.

I don't think it's possible to capture UAC prompts on WGC even if we did run it as a service.

However UAC is not needed if sunshine is running as admin and the user as selected that their app runs as administrator in the configuration.

**Comment by TannicArturo98** [comment]
I've the same issue with "Run as admin" checkbox checked in WebUI and Sunshine ran as admin

**Comment by ReenigneArcher** [comment]
> I've the same issue with "Run as admin" checkbox checked in WebUI and Sunshine ran as admin

UAC is visible if you use the service (default with our Windows installer) and DXGI (instead of Windows.Graphics.Capture).

**Comment by TannicArturo98** [comment]
> > I've the same issue with "Run as admin" checkbox checked in WebUI and Sunshine ran as admin
> 
> UAC is visible if you use the service (default with our Windows installer) and DXGI (instead of Windows.Graphics.Capture).

I've installed latest version of Sunshine Windows_AMD_64.exe but it doesn't show me the UAC prompt when i try to run a program as admin.

**Comment by ReenigneArcher** [comment]
Yes, because you need to use the "service" and not run as admin.
